projects/habitat_ovmm/utils/metrics_utils.py

```diff
@@ -56,7 +56,7 @@ def compute_stats(aggregated_metrics: pd.DataFrame) -> dict:
 
     # find indices in the DataFrame with stage success in their name and compute success rate
     for k in aggregated_metrics.index:
-        if ("phase_success" in k and "END" in k) or "task_success" in k:
+        if ("phase_success" in k and "END" in k) or "overall_success" in k:
             stats[k.replace("END.ovmm_", "")] = aggregated_metrics.loc[k]["mean"]
 
     stats["partial_success"] = aggregated_metrics.loc["partial_success"]["mean"]
```

Comment:
partial_success -> rearrangement_progress?

Reply:
I created this other PR: https://github.com/facebookmicrosites/habitat-website/pull/177 to keep it consistent with our paper